x-pack/plugins/aiops/public/components/change_point_detection/fields_config.tsx

```diff
@@ -20,6 +20,10 @@ import {
 } from '@elastic/eui';
 import { FormattedMessage } from '@kbn/i18n-react';
 import { i18n } from '@kbn/i18n';
+import { FieldStatsServices } from '@kbn/unified-field-list-plugin/public';
```

Comment:
can be import type FieldStatsServices here

Reply:
updated in https://github.com/elastic/kibana/pull/155177/commits/eb9f239ea957e443e1bd3336ecfd8dbc9b5833e4